tests/ocf-meta-inf_manifest/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,8 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>Epub with manifest file inside meta-inf</title>
+  </head>
+  <body>
+    <p>Test passes if only can access this page of the book</p>
```

Comment:
I do not really understand the sentence. I presume this text should be

> Test passes if only this page of the book is accessible by the reader

But I would prefer @dlazin to be the final arbiter on linguistic issues... :-)

Reply:
Yeah, that's exactly what I meant. Thanks @iherman. Updated.